### PR TITLE
Fixed Spellcheck errors and removed Broken link

### PR DESCRIPTION
**Pull request by ChetanVyas**
- Fixed Spellcheck errors in "\governance\related-groups\README.md".
- removed broken link - "https://standards.ieee.org/develop/project/1915.1.html".

### CODE REVIEW

File: governance/related-groups/README.md [hunk 1]
```diff
@@ -16,29 +16,29 @@ the work of Security TAG
...
 * [Cloud Security Alliance Working Groups](https://cloudsecurityalliance.org/research/working-groups/)
-  * Security TAG members: @achetal01
+<!-- cSpell:ignore achetal01-->  * Security TAG members: @achetal01
 
 * [IEEE 7009](https://standards.ieee.org/develop/project/7009.html): Standard
   for Fail-Safe Design of Autonomous and Semi-Autonomous Systems Working Group
   * Security TAG members:
 
-* [IEEE P1915.1](https://standards.ieee.org/develop/project/1915.1.html):
+* [IEEE P1915.1] :
```
Comment (ultrasaurus) [inline governance/related-groups/README.md:41]: I think this change would leave a broken link without the warning, not actually removing the link.

However, I suggest using Internet Archive to refer to the archived version of the page at: https://web.archive.org/web/20180729062624/https://standards.ieee.org/develop/project/1915.1.html

Comment (lumjjb) [reply governance/related-groups/README.md:41]: My thought was to remove the whole link, but i like the archive link idea, am in favor of using the archive as well (with an add a warning to it that it is using archive).

we could then add an issue to address it more permanently.

File: governance/related-groups/README.md [hunk 2]
```diff
@@ -16,29 +16,29 @@ the work of Security TAG
     - [Storage](https://github.com/cncf/tag-app-delivery) -
 
 * [CNCF TOC](https://github.com/cncf/toc): Technical Oversight Committee
-  * Security TAG members: @justincormack , @lizrice
+<!-- cSpell:ignore justincormack, lizrice-->  * Security TAG members: @justincormack , @lizrice
```
Comment (lumjjb) [inline governance/related-groups/README.md:19]:
```suggestion
<!-- cSpell:ignore justincormack, lizrice-->
  * Security TAG members: @justincormack , @lizrice
```

Can we make these like this instead? easier to see the formating.

Comment (ChetanVyas) [reply governance/related-groups/README.md:19]: Sure, should I add the archive link also?

Comment (lumjjb) [reply governance/related-groups/README.md:19]: yea lets do that

Comment (ChetanVyas) [reply governance/related-groups/README.md:19]: * [IEEE P1915.1](https://web.archive.org/web/20180729062624/https://standards.ieee.org/develop/project/1915.1.html) (Warning: Archive Link):
Is this right?

Comment (lumjjb) [reply governance/related-groups/README.md:19]: looks right!

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

**Comment by lumjjb** [comment]
Thanks @ChetanVyas ! Remember to add yourself to the members list in the README.md if you haven't already!

**Pull request merged**